scss/theme.scss

```diff
@@ -0,0 +1,99 @@
+@mixin theme-colored($color1, $color2, $color3) {
+
+  // Links
+  a {
+    color: $color1;
+
+    &:hover,
+    &:focus {
+      color: $color1;
+    }
+  }
+
+  // Buttons
+  .btn-default,
+  a.btn-default {
+    @include button-colored(#e9e9e9, #ccc, $base-text-color);
+  }
+  .btn-one,
+  a.btn-one {
+    @include button-colored($color1, $color1 - #151515, white);
+  }
+
+  .btn-two,
+  a.btn-two {
+    @include button-colored($color2, $color2 - #151515, white);
+  }
+
+  .btn-three,
+  a.btn-three {
+    @include button-colored($color3, $color3 - #151515, white);
+  }
+
+  // forms
+  textarea:focus,
+  .textarea:focus,
+  input[type="text"]:focus,
+  input[type="password"]:focus,
+  input[type="email"]:focus,
+  input[type="number"]:focus,
+  select:focus {
+    border: 1px solid $color1;
+  }
+
+  // sections
+  .application-header {
+    background-color: $color1;
+    textarea,
```

Comment:
Rule declaration should be preceded by an empty line<br>Avoid qualifying attribute selectors with an element.